src/libs/zuora.ts

```diff
@@ -133,10 +133,19 @@ Additional Comms                                                   # reserved fo
     FROM
       RatePlanCharge
     WHERE
-      Product.ProductType__c = 'Newspaper - National Delivery' 
-      and RatePlanCharge.name = '${dayOfTheWeekName}' 
-      and RatePlanCharge.effectiveStartDate <= '${date}'
-      and (RatePlanCharge.effectiveEndDate >= '${date}' or (Subscription.autoRenew = true and Subscription.status = 'Active'))
+      (Subscription.Status = 'Active' OR Subscription.Status = 'Cancelled') AND
+      Product.Name = 'Newspaper - National Delivery' AND
+      RatePlanCharge.name = '${dayOfTheWeekName}' AND
+      RatePlanCharge.effectiveStartDate <= '${date}' AND
+      (
+          RatePlanCharge.effectiveEndDate > '${date}' OR
```

Comment:
[In Zuora, dates change at midnight (12:00 AM),](https://knowledgecenter.zuora.com/Zuora_Central_Platform/Tenant_Management/Dates_in_Zuora/B_Date_and_dateTime_Formats#:~:text=In%20Zuora%2C%20dates%20change%20at%20midnight%20(12%3A00%20AM)%2C) so we have to exclude today